.github/pull_request_template.md

```diff
@@ -86,7 +86,9 @@ Note that GitHub prefixes anchor names in markdown with "user-content-".
 
 #### Should this PR be included in the next patch release?
 
+`Yes` should be selected for bug fixes, documentation updates, and other small changes. `No` should be selected for new features and larger changes or if there is no recent minor/patch release. If you're unsure about the release classification of this PR, leave this unchecked to let the maintainers decide.
```

Comment:
Can we define "minor release" and "patch release"?

Reply:
@dbczumar Added https://github.com/mlflow/mlflow/pull/11746/files#diff-b2496e80299b8c3150b1944450bd81c622e04e13d15c411d291db0927d75fd6bR91-R98